cpu/atxmega/include/cpu_ebi.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_atxmega
+ * @{
+ *
+ * @file
+ * @brief           External Bus Interface API
+ *
+ * @author          Gerson Fernando Budke <nandojve@gmail.com>
+ */
+
+#include "periph_cpu.h"
+
+#ifndef CPU_EBI_H
+#define CPU_EBI_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef uint32_t hugemem_ptr_t;
+
+#define HUGEMEM_NULL        0
+
+void ebi_init(void);
+
+static inline uint_fast8_t hugemem_read8(const hugemem_ptr_t from)
```

Comment:
```suggestion
static inline uint8_t hugemem_read8(const hugemem_ptr_t from)
```

On AVR, the `uint_fast8_t`, `uint_fast16_t`, ... are identical to `uint8_t`, `uint16_t`, ... So why not safe a few chars to type here?

Doxygen is missing.

Reply:
Done!